dist/gmusic.js

```diff
@@ -250,7 +250,7 @@ proto.rating = {
     // Selected thumbs up:
     // <paper-icon-button icon="sj:thumb-up-outline" data-rating="5" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0" title="Undo thumb-up" aria-label="Undo thumb-up"></paper-icon-button>
     // jscs:enable maximumLineLength
-    return el.getAttribute('aria-label').indexOf('Undo') !== -1;
+    return el.__data__.icon === 'thumb-up' || el.__data__.icon === 'thumb-down';
```

Comment:
It's a polymer thing, it's only present on polymer elements and isn't releated to data attributes